src/main/java/com/xceptance/xlt/report/util/ReportUtils.java

```diff
@@ -338,4 +340,24 @@ public static String obtainProjectName(final Iterable<Document> documents)
         }
         return StringUtils.defaultString(projectName);
     }
+
+    /**
+     * Calculates the percentage given by count/total, as a {@link BigDecimal} (rounding it to have at most
+     * {@value #DEFAULT_DECIMAL_PLACES} decimal places). Returns 0 if total is 0.
+     * 
+     * @param count
+     * @param total
+     * @return
+     */
+    public static BigDecimal calculatePercentage(int count, int total)
+    {
+        if (count == 0)
```

Comment:
Should be `total` instead of `count`, right?

Reply:
:facepalm: